classes/article/ArticleGalleyDAO.inc.php

```diff
@@ -142,10 +140,8 @@ function getByContextId($journalId) {
 			FROM	publication_galleys g
 				INNER JOIN publications p ON (p.publication_id = g.publication_id)
 				LEFT JOIN submissions s ON (s.submission_id = p.submission_id)
-				LEFT JOIN submission_files sf ON (g.file_id = sf.file_id)
-				LEFT JOIN submission_files nsf ON (nsf.file_id = g.file_id AND nsf.revision > sf.revision)
```

Comment:
Are you sure you want to remove this join? It ensures that only the latest `submission_files sf` is joined, rather than all.

Reply:
Yes, I think so. There is no longer more than one row per `submission_file_id`, so there is no need to fetch the "latest".

Reply:
Great! And good riddance to that clever bit of SQL, as fancy as it was.